### PR TITLE
Add asset must (cometh)

### PR DESCRIPTION
**Pull request by CPatchane**
https://www.coingecko.com/en/coins/must

### CODE REVIEW

File: rotkehlchen/data/all_assets.json [hunk 1]
```diff
@@ -7377,6 +7377,16 @@
         "symbol": "MCI",
         "type": "ethereum token"
     },
+    "MUST": {
+        "coingecko": "Must",
+        "cryptocompare": "",
+        "ethereum_address": "0x9c78ee466d6cb57a4d01fd887d2b5dfb2d46288f",
```
Comment (LefterisJP) [inline rotkehlchen/data/all_assets.json:7383]: The addresses must always be checksummed. Tests will thankfully fail and won't let you commit that.

File: rotkehlchen/data/all_assets.json [hunk 2]
```diff
@@ -7377,6 +7377,16 @@
...
+        "ethereum_token_decimals": 18,
+        "name": "MUST (Cometh)",
+        "started": 1610699161,
```
Comment (LefterisJP) [inline rotkehlchen/data/all_assets.json:7386]: Where is this timestamp coming from? Checking the [transaction](https://etherscan.io/tx/0x394e686870eccb31abf47f91d552e6b2013094d85439030252dc0c1f42baaa65) the contract was deployed the timestamp at time of deployment should be 1610742361 right?

**Review by LefterisJP — COMMENTED** [review]
Hey @CPatchane this will need some more changes as per my comments.

Also you will need to change the md5 hash of all_assets.meta. I think this should also be mentioned in the docs. Finally adjust the test_assets.py test to make sure that the new hash matches: https://github.com/rotki/rotki/blob/80893e93a9b2e74287a5949c5fb742b5a068cecc/rotkehlchen/tests/unit/test_assets.py#L266-L283. This is the hash of all_assets.json.

File: rotkehlchen/data/all_assets.json (hunk 2 above)
Comment (CPatchane) [reply rotkehlchen/data/all_assets.json:7386]: You're right? I did for 8AM instead of 8PM 🤦

**Review by LefterisJP — APPROVED** [review]
LGTM. There is some docs errors and typos which I will fix in a new PR.

**Pull request merged**